.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+on:
+  - pull_request
+jobs:
+  install_formula:
+    runs-on: macos-11
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v2
+      - 
+        name: Install rainforest-cli
+        run: |
+          brew install --formula ./Formula/rainforest-cli.rb
+      - 
+        name: Check rainforest works
+        env:
+          URL: ${{ github.event.pull_request.comments_url }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          VERSION=`rainforest --version --skip-update`
+          echo $VERSION
+          curl \
+            -X POST \
+            $URL \
+            -H "Content-Type: application/json" \
+            -H "Authorization: token $GITHUB_TOKEN" \
+            --data "{ \"body\": \"This update reported it's version as \`$VERSION\` 🤞\" }"
```

Comment:
just a grammar nit - there shouldn't be an apostrophe in `its`

Reply:
😊